views/mailinglists.ejs

```diff
@@ -2,66 +2,17 @@
 <% title = 'OpenMRS ID - Mailing Lists' %>
 <% headline = 'Mailing Lists' %>
 <% headAppend = '<link rel="stylesheet" href="/mailinglists/resource/mailinglists.css">' %>
-<% sidebar = ['./../views/sidebar/groups-secondaryemail'] %>
```

Comment:
Resolved in discussion on IRC - Using 2 column layout to prevent sidebar from going to the bottom of the page.